code/index.html

```diff
@@ -1,18 +1,133 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <title>Project Name</title>
-    <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
-  </head>
-  <body>
-    <header>
-      <!-- If you want a header with a title, logo and a navbar, here is where you should add that. -->
-    </header>
-
-    <main class="">
-      <!-- The main container to use as the flex parent for your card layout, the main content of the site should be in here.  -->
-    </main>
-  </body>
-</html>
+
+<head>
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Textildesign</title>
+  <link rel="stylesheet" href="./style.css" />
+  <!-- dont forget the meta tag for responsiveness -->
+</head>
+
+<body>
+  <header>
+    <div class="hamburger">
+      <div></div>
+      <div></div>
+      <div></div>
+    </div>
+    <div class="header-text">
+      <h1>Textildesign</h1>
+      <h2>Anna Leutert</h2>
+    </div>
+  </header>
+
+  <nav id="navigation">
+    <ul>
+      <li><a class="active" href="index.html">Home</a></li>
+      <li><a href="about.html">Über Mich</a></li>
+      <li><a href="aktuell.html">Aktuell</a></li>
+      <li><a href="portfolio.html">Portfolio</a></li>
+      <li><a href="blog.html">Tipps & Tricks</a></li>
+    </ul>
+  </nav>
+
+  <main class="">
+
+    <!--über mich / vorstellen -->
+
+    <section class="welcome-container">
+      <section class="text-container">
+        <h3>Mit viel Begeisterung weben.</h3>
+        <p>Mein ganzes Leben schon bin ich fasziniert von bunten Fäden und was daraus gemacht werden kann. Ich fühle
+          mich
+          wohl beim Weben, das ist meine Welt.</p>
+        <a href="about.html" class="button"> Mehr zu meiner Person</a>
+      </section>
+      <div class="picture">
+        <img src="/img/portrait.JPG" alt="portrait" style="width:100%;max-width:500px;">
```

Comment:
To reach the images more easily, you could add your image-folder to the code-folder. The path you've used now suggests that the img-folder is located within the code-folder, if you want to reach a folder outside of the code-folder, you would need to write `../img/portrait.JPG` (with 2 dots before the slash).  😊